app/components/recirculation-prefooter.js

```diff
@@ -127,7 +147,12 @@ export default Component.extend(
       this.fetch.fetchFromMediawiki(url, TopArticlesFetchError)
         .then((data) => {
           this.set('topArticles', data.slice(0, 3));
-          this.set('items', data.slice(3));
+
+          if (this.shouldUseExperimentalService) {
+            this.set('fallbackItems', data.slice(3));
```

Comment:
Why do we need a distinction between items and fallback items?

Reply:
`fallbackItems` are set in case `recommendedData` is fetched, because `recommended_articles` are set as `items` then. I wanted to explicitly state that they will be used as a `plan B`